cpanfile

```diff
@@ -119,6 +120,7 @@ author_requires 'Test::EOL';
 author_requires 'Test::NoTabs';
 
 test_requires 'Cache::Null';
+test_requires 'Catalyst::Plugin::Static::Simple';
```

Comment:
This is pulled in by `Catalyst::Devel`, but that module is only `author_requires`.